gui/default/syncthing/core/localeNumberFilter.js

```diff
@@ -1,9 +1,6 @@
 angular.module('syncthing.core')
     .filter('localeNumber', function () {
-        return function (input, decimals) {
-            if (typeof(decimals) !== 'undefined') {
-                return input.toLocaleString(undefined, {maximumFractionDigits: decimals});
-            }
+        return function (input) {
             return input.toLocaleString();
         };
-    });
\ No newline at end of file
+    });
```

Comment:
Can't we just call `{{ x.toLocaleString() }}` ?
This doesn't seem to do much?

Reply:
It doesn't, but it is chained with `| alwaysNumber` which isn't compatible with `.toLocaleString`(?)